lib/oxidized/hook/githubrepo.rb

```diff
@@ -44,6 +44,10 @@ def fetch_and_merge_remote(repo)
 
   private
 
+  def certificate_validation
+    lambda { |valid, host| @cfg.secure == false ? true : valid }
```

Comment:
Should use `cfg` rather than `@cfg` to be consistent with the code in all existing hooks.

Would be cleaner to explicitly check if the key exists as well, i.e. 
```
cfg.has_key? 'secure' and cfg.secure == false ? true : valid
```
rather than just
```
cfg.secure == false ? true : valid
```

Reply:
These changes make perfectly sense for me. I will apply them right away.